### ISSUE TITLE
Songs panel on the local library when on the 3rd view option (folder icon) glitches

### ISSUE THREAD

**Issue by Kritsos** [nukeop/nuclear#1138]
**Platform:**
Windows 10
Version: 20H2
OS Build: 19042.1415

**Nuclear version:**
0.6.17

**Description of the issue:**
When on the local library using the 3rd view option (the one with the folder icon) if I expand my folders until a scroll bar is needed, the panel glitches out. A grey line on the bottom of the panel starts flashing and when I click on a song it doesn't get selected. Only after I have clicked many times does the flashing stop and the songs can be selected normally.

Steps to reproduce the bug:
1. Go to the local library and select the 3rd view option, the one with the folder icon.
2. Expand the folders until a scroll bar is needed.

Below is a showcase of the bug. In the recording both the scroll bar and the grey bottom line are flashing, however when I was reproducing the bug in my screen only the grey bottom line was flashing and not as intensely.

https://user-images.githubusercontent.com/72339454/148950850-c4b71d23-0abf-48bc-9f8a-813ba7fa0cd3.mp4

**Comment by Venryx** [comment]
Just wanted to mention that I'm the one who originally contributed that third view option, and the scrollbar glitching issue was not present at that time. The issue came about some weeks/months later, when I downloaded a new version (after various UI changes had been made) -- so it is due to a regression of some sort. (I do not have time to debug it atm unfortunately)

**Comment by nukeop** [comment]
It must be this file: https://github.com/nukeop/nuclear/blob/6a99b3f85042f170de840d3e9b9381d8cd5fd84b/packages/app/app/components/LibraryView/LibraryFolderTree/index.js

If it was not present at that time, then it could have been broken because of upgraded libraries (notably Electron) - hard to test against such graphical issues.
I remember a similar issue being caused somewhere by using an estimate of a row's height/width. I'd also look at the `AutoResizer` component. I'll try to find the time to reproduce and debug this more thoroughly.

**Comment by Venryx** [comment]
Yes; I don't see any changes in the commit after mine (adding the feature) which seems likely to have caused the graphical glitching.

So my guess is either that one of the dependencies involved was updated (to a version that conflicted with the layout approach used), or that something about the surrounding UI elements was changed that "manifested" an issue that had previously just not been noticed. (for example, if the CSS for scrollbar styling was changed subtly or something)

**Comment by nukeop** [comment]
This issue is from an older version of Nuclear that has been replaced with a complete rewrite. None of the old issues apply to the new codebase.

The old code is preserved on the [legacy/electron](https://github.com/nukeop/nuclear/tree/legacy/electron) branch. If you have feedback about the new version, head to [Discussions](https://github.com/nukeop/nuclear/discussions).
